sim/mcuboot-sys/build.rs

```diff
@@ -0,0 +1,94 @@
+// Build mcuboot as a library, based on the requested features.
+
+extern crate gcc;
+
+use std::env;
+use std::fs;
+use std::io;
+use std::path::Path;
+
+fn main() {
+    // Feature flags.
+    let sig_rsa = env::var("CARGO_FEATURE_SIG_RSA").is_ok();
+    let sig_ecdsa = env::var("CARGO_FEATURE_SIG_ECDSA").is_ok();
+    let overwrite_only = env::var("CARGO_FEATURE_OVERWRITE_ONLY").is_ok();
+
+    // TODO: Force clang if we are requestion fuzzing.
```

Comment:
Other than the 'requestion' word. But, I don't really need the comment. When I use libfuzzer it should be obvious that it needs to build with clang, since the fuzzing arguments are clang only.